src/bundle/Resources/views/content/form_fields.html.twig

```diff
@@ -82,7 +82,12 @@
     {% set attr = attr|merge({'class': (attr.class|default('') ~ ' ez-data-source__input')|trim}) %}
     {% set wrapper_attr = wrapper_attr|default({})|merge({'class': (wrapper_attr.class|default('') ~ ' ' ~ wrapper_class)|trim}) %}
 
-    <div{% with { attr: wrapper_attr } %}{{ block('attributes') }}{% endwith %}>
+    {% if fieldtype.vars.value.fieldDefinition.descriptions[fieldtype.vars.languageCode] is defined %}
```

Comment:
For me it looks more readable after extracting `field type.vars.value.field Definition.descriptions` to the local variable:

```twig
{% set field_type_descriptions = fieldtype.vars.value.fieldDefinition.descriptions %}
{% if field_type_descriptions[fieldtype.vars.languageCode] is defined %}
    {% set wrapper_attr = wrapper_attr|merge({'title': field_type_descriptions[fieldtype.vars.languageCode]}) %}
{% endif %}
```

Reply:
@ViniTou could you incorporate this feedback?

Reply:
@lserwatka 
done